pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.wavefront</groupId>
-    <artifactId>dropwizard-metrics-sdk</artifactId>
+    <artifactId>wavefront-dropwizard-metrics-sdk-java</artifactId>
     <version>0.9.1-SNAPSHOT</version>
```

Comment:
Should it be reverted back to 0.9.0-SNAPSHOT.

Reply:
I think the version here is for the next development version, so it should be `0.9.1-SNAPSHOT`

Reply:
I reverted this back to 0.9.0 given we are changing the artifact name